tests/humanize_tests/tests.py

```diff
@@ -70,6 +73,9 @@ def test_ordinal(self):
             "102nd",
             "103rd",
             "111th",
+            "0th",
```

Comment:
I don't agree with that. I do think there are use cases for 0th. For example: the 0th law of thermodynamics. It may not be common, but it is valid usage.

Reply:
I agree with @Siburg: "0th" is uncommon but correct.

Reply:
Alright, as we already have 0 -> 0th let's keep it in case someone's using it.